pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -144,6 +143,40 @@ public Policies getPolicies(@PathParam("property") String property, @PathParam("
         return getNamespacePolicies(namespaceName);
     }
 
+    @GET
+    @Path("/{property}/{cluster}/{namespace}/policies/{policyProperty}")
```

Comment:
We should add these handlers also in the v2 API. 

Additionally, in the v2 API we can remove all the other specific get/set handlers.